apps/portals/src/portal-components/nf/style/_BrowseToolsPage.scss

```diff
@@ -68,5 +65,8 @@
   .highlightSubmitToolButton {
     background-color: #e27134;
     color: white;
+    &:hover {
+      background-color: #dc6525;
+    }
```

Comment:
Add hover state to match other buttons -- since this isn't a standard MUI button, I calculated the hover color using the same rgb difference as the "View All Tools" button on the same page. Happy to adjust if there's a better way to get the hover color!

main

https://user-images.githubusercontent.com/26949006/235273211-784660cd-4f5c-44ae-8a7c-5b191e2ce036.mov

feature

https://user-images.githubusercontent.com/26949006/235273214-84ce2aad-912b-4984-b818-32b327a97198.mov

Reply:
Got it, that makes sense about the other states. As far as I can tell, the background color only changes on hover for a MUI button with a standard color (e.g. primary), so I think just setting the hover state handles this case. Happy to also set styles for the active and focus states, if I've missed something though!

I've also opened a [ticket](https://sagebionetworks.jira.com/browse/PORTALS-2653) to track the theme issue and added a TODO comment to this section, so it can be cleaned up later